lib/puppet-strings/yard/handlers/ruby/base.rb

```diff
@@ -18,7 +18,13 @@ def node_as_string(node)
     when :label
       node.source[0..-2]
     when :dyna_symbol
-      node.source
+      # YARD 0.9.20 changed how dyna_symbols are represented
+      # https://github.com/lsegal/yard/commit/225ded9ef38c6d2be5a3b0fc7effbc7d6644768d
+      if yard_version >= Gem::Version.new('0.9.20')
+        node.source[2..-2]
```

Comment:
This seems like a really fragile way to do this, but I trust your judgement that there's not a better way.

Reply:
Yeah I wasn't super happy with it.  But it's no worse than the examples above or below 🤷‍♂️ The fragility is also why I add tests so at least we should be able to detect when it goes bad.